### PR TITLE
Add some audit scripts

### PR DESCRIPTION
**Pull request by bengerman13**
This adds two scripts that might be useful for future auditing:
- bosh-audit.py gets bosh events with optional time and user filters
- cf-audit.py gets cloudfoundry events with optional time and user filters

### CODE REVIEW

File: bosh-audit.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
```
Comment (hillaryj) [inline bosh-audit.py]: Could we add an overview docstring with the description from the PR here?

Comment (hillaryj) [reply bosh-audit.py]: A description is in the ArgParser description but would be nice to overview for future devs

File: cf-audit.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
```
Comment (hillaryj) [inline cf-audit.py]: Overview docstring here like in bosh-audit.py?

File: bosh-audit.py (hunk 1 above)
Comment (bengerman13) [reply bosh-audit.py]: that seems smart

Comment (mxplusb) [reply bosh-audit.py]: Can you append that to the argparse help? Since you're at it...

Comment (bengerman13) [reply bosh-audit.py]: Done

**Pull request merged**